src/modules/main/sidebar/Sidebar.tsx

```diff
@@ -37,13 +40,20 @@ export const Sidebar = ({ basePath }: SidebarProps) => {
 
   const activeLinkIdx = getActiveLinkIdx(pathname.replace(basePath, ''), sidebarLinks);
 
+  const [open, setOpen] = useState(false);
```

Comment:
open, setOpen use state should be at the beginning of the component

Reply:
useState has been moved